qa/rolling-upgrade/src/test/java/org/opensearch/upgrades/IndexingIT.java

```diff
@@ -145,6 +147,18 @@ private void waitForClusterHealthWithNoShardMigration(String indexName, String s
         client().performRequest(waitForStatus);
     }
 
+    private void verifySegmentStats(String indexName) throws Exception {
+        assertBusy(() -> {
+            Request segrepStatsRequest = new Request("GET", "/_cat/segment_replication/" + indexName);
```

Comment:
Can we use the client here so we don't need to manually parse the _cat response?

Reply:
@mch2 : Sorry, I did not understand. Can you please add more details or provide example ?

Reply:
@mch2 : Resolving this comment for now, please feel free to unresolve.